packages/ra-ui-materialui/src/field/ReferenceField.js

```diff
@@ -36,11 +35,9 @@ export const ReferenceFieldView = ({
 
     if (resourceLinkPath) {
         return (
-            <Link
-                className={classnames(classes.link, className)}
-                to={resourceLinkPath}
-            >
+            <Link to={resourceLinkPath} className={className}>
                 {React.cloneElement(children, {
+                    className: classes.link, // force color override for Typography components
```

Comment:
or should I apply the `className` prop to the child, too? I'm afraid it's a BC break.